mmengine/visualization/visualizer.py

```diff
@@ -862,6 +1089,8 @@ def draw_binary_masks(
             f'but got {binary_masks.dtype}')
         binary_masks = binary_masks.astype('uint8') * 255
         img = self.get_image()
+        if self.backend == 'cv2':
+            img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
```

Comment:
If  the conversion `BGR2RGB` is done here, and users directly save the image by:

```python
visualizer.draw_binary_masks(...)
img = visualizer.get_image()
cv2.imwrite('path', img)
```

It seems that the image will be saved in a `BGR` format since that `opencv` cast will the channel order `BGR` to `RGB` during saving.